app/src/components/layout/Header.tsx

```diff
@@ -224,44 +230,48 @@ const Header: React.FC = () => {
           </Container>
         </Toolbar>
 
-        <AuthGuard>
-          <Box className={classes.mainNav}>
-            <Container maxWidth="xl">
-              <Toolbar
-                variant="dense"
-                disableGutters
-                className={classes.mainNavToolbar}
-                role="navigation"
-                aria-label="Main Navigation">
+        <Box className={classes.mainNav}>
+          <Container maxWidth="xl">
+            <Toolbar
+              variant="dense"
+              disableGutters
+              className={classes.mainNavToolbar}
+              role="navigation"
+              aria-label="Main Navigation">
+              <UnAuthGuard>
+                <RouterLink to="/" id="menu_home">
+                  Home
+                </RouterLink>
+              </UnAuthGuard>
+              <AuthGuard>
                 <RouterLink to="/admin/projects" id="menu_projects">
-                  Projects
+                  My Projects
                 </RouterLink>
-                <SystemRoleGuard validSystemRoles={[SYSTEM_ROLE.DATA_ADMINISTRATOR, SYSTEM_ROLE.SYSTEM_ADMIN]}>
-                  <RouterLink to="/admin/search" id="menu_search">
-                    Map
-                  </RouterLink>
-                </SystemRoleGuard>
-                <RouterLink to="/admin/resources" id="menu_resources">
-                  Resources
+              </AuthGuard>
+              <SystemRoleGuard validSystemRoles={[SYSTEM_ROLE.DATA_ADMINISTRATOR, SYSTEM_ROLE.SYSTEM_ADMIN]}>
+                <RouterLink to="/admin/search" id="menu_search">
+                  Map
                 </RouterLink>
-                <SystemRoleGuard validSystemRoles={[SYSTEM_ROLE.SYSTEM_ADMIN, SYSTEM_ROLE.DATA_ADMINISTRATOR]}>
-                  <RouterLink to="/admin/users" id="menu_admin_users">
-                    Manage Users
-                  </RouterLink>
-                </SystemRoleGuard>
-              </Toolbar>
-            </Container>
-          </Box>
-        </AuthGuard>
+              </SystemRoleGuard>
+              <RouterLink to="/resources" id="menu_resources">
+                Resources
+              </RouterLink>
+              <SystemRoleGuard validSystemRoles={[SYSTEM_ROLE.SYSTEM_ADMIN, SYSTEM_ROLE.DATA_ADMINISTRATOR]}>
+                <RouterLink to="/admin/users" id="menu_admin_users">
+                  Manage Users
+                </RouterLink>
+              </SystemRoleGuard>
+            </Toolbar>
+          </Container>
+        </Box>
       </AppBar>
 
       <Dialog open={open}>
         <DialogTitle>Contact Support</DialogTitle>
         <DialogContent>
           <Typography variant="body1" component="div" color="textSecondary">
-            For technical support or questions about this application, please email&nbsp;
+            For technical support or questions about this application, please email &zwnj;
```

Comment:
Why the change here? Just curious.

Reply:
Just to avoid the nagging `{' '}` literal